src/apps/js_generic/js_generic.cpp

```diff
@@ -80,6 +81,31 @@ namespace ccfapp
     JS_FreeValue(ctx, exception_val);
   }
 
+  static JSValue js_generate_aes_key(
+    JSContext* ctx, JSValueConst, int argc, JSValueConst* argv)
+  {
+    if (argc != 1)
+      return JS_ThrowTypeError(
+        ctx, "Passed %d arguments, but expected 1", argc);
+
+    int32_t key_size;
+    if (JS_ToInt32(ctx, &key_size, argv[0]) < 0)
+    {
+      js_dump_error(ctx);
+      return JS_EXCEPTION;
+    }
+    if (key_size != 128 && key_size != 192 && key_size != 256)
```

Comment:
Comment explaining that these are the supported key size for AES-128.

Reply:
Done.

Reply:
Thank you!